IOSDeviceLib/IOSDeviceLib.cpp

```diff
@@ -286,13 +287,17 @@ void on_device_found(const DevicePointer* device_ptr, std::string device_identif
 		}
 		
 		update_device_result(device_identifier, result);
+        if (result["deviceClass"] == "AppleTV") {
+            // We do not support AppleTV devices
+            result = nullptr;
+        }
 	}
 }
 
 void device_notification_callback(const DevicePointer* device_ptr)
 {
 	std::string device_identifier = get_cstring_from_cfstring(AMDeviceCopyDeviceIdentifier(device_ptr->device_info));
-	json result;
+	json result = nullptr;
 	result[kDeviceId] = device_identifier;
```

Comment:
Not sure this is correct.

Reply:
Me too, but it's working. It should be the same. The updated version is more clear, as we are comparing with `nullptr` below.